vault/resource_secrets_sync_aws_destination.go

```diff
@@ -86,6 +88,12 @@ func awsSecretsSyncDestinationResource() *schema.Resource {
 				Optional:    true,
 				Description: "Extra protection that must match the trust policy granting access to the AWS IAM role ARN.",
 			},
+			consts.FieldGranularity: {
+				Type:     schema.TypeString,
+				Optional: true,
+				Description: "Determines what level of information is synced as a distinct resource at the destination. " +
+					"Can be 'secret-path' or 'secret-key'",
```

Comment:
This whole description could be a const since it's used in all destination types.

Reply:
Good call, moved this schema field into the common schema fields so that there is only one source of truth. Thanks!